test/TestAsset/LowerCase.php

```diff
@@ -4,15 +4,27 @@
 
 namespace LaminasTest\Filter\TestAsset;
 
-use Laminas\Filter\AbstractFilter;
+use Laminas\Filter\FilterInterface;
 
+use function is_string;
 use function strtolower;
 
-/** @template-extends AbstractFilter<array{}> */
-class LowerCase extends AbstractFilter
+/** @implements FilterInterface<string> */
```

Comment:
I guess the type is here so that we tell what is being return from `filter` function?
If I see correct, `filter` returns `mixed` here as there is `! is_string` check which returns the value without touching it?